src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+        }
+        printDashLine();
+        System.out.printf("Hello! I'm Duke, your task manager.%n" +
+                "Its the first time I see you%n" +
+                "Key in your tasks below!%n");
+        printDashLine();
+    }
+
+    static void load() {
+        printDashLine();
+        System.out.printf("Hello! I'm Duke, your task manager.%n" +
+                "I have loaded the tasks you have keyed in the last time%n" +
+                "Continue to key in your tasks below!%n");
+        printDashLine();
+        taskBank.loadTasks();
+    }
+
+    static void talk(Scanner sc) {
+        String sentence;
+        while (true) {
+            try {
+                sentence = sc.nextLine();
+                if (sentence.isEmpty()) {
+                    throw new EmptyInputException("Empty input! Try again (o|o)\n");
+                } else if (sentence.equals("bye")) {
+                    Chatter.bye();
+                    break;
+                } else if (sentence.equals("list")) {
+                    Chatter.list();
+                } else if (sentence.startsWith("done")) {
+                    Chatter.done(sentence);
+                } else if (sentence.startsWith("todo")) {
+                    Task newTask = Chatter.taskBank.addTodo(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("deadline")) {
+                    Task newTask = Chatter.taskBank.addDeadline(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("event")) {
+                    Task newTask = Chatter.taskBank.addEvent(sentence);
+                    Chatter.add(newTask);
+                } else if (sentence.startsWith("delete")) {
+                    Chatter.deleteTask(sentence);
+                } else if (sentence.startsWith("clear")) {
+                    Chatter.clear();
+                } else {
+                    throw new IrregularInputException("☹ OOPS!!! I'm sorry, but I don't know what that means :-(" +
+                            "\nPlease try again!");
+                }
+            } catch (IrregularInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (RepeatedCompletionException e) {
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (EmptyInputException e) {
+                printDashLine();
+                System.out.println(e.getMessage());
+                printDashLine();
+            } catch (NumberFormatException e) {
+                printDashLine();
+                System.out.printf("Not a valid number. Try again%n");
+                printDashLine();
+            }
+        }
+    }
```

Comment:
You should minimize the size of the `talk` method by splitting the different functionalities into their own methods. By reducing the size of the method and ensuring each method does one functionality it is supposed to do, your code is easier to maintain, scale and debug.